static/src/javascripts/boot.js

```diff
@@ -70,9 +70,15 @@ define([
 
 
 
-    var shouldRunEnhance = guardian.isModernBrowser;
+    var shouldRunEnhance = guardian.isEnhanced;
+    var shouldRunCommercial = guardian.isEnhanced;
```

Comment:
You wrote "Our commercial IE8 check uses `isModernBrowser`." Shouldn't this rather be `guardian.isModernBrowser`?